### PR TITLE
Modify Placement json marshalling

### PR DESCRIPTION
**Pull request by cw9**
@xichen2020 
Modify the json marshalling for placement snapshot, I'll remove the multi-cluster placement in statsdex. Will wait until the statsdex change is done to merge this in.

### CODE REVIEW

**Comment by xichen2020** [comment]
Can we rename the PR perhaps for documentation purposes?

**Comment by cw9** [comment]
Sure yeah, didn't know how did that end up being the title, modified.

File: placement/options.go [hunk 1]
```diff
@@ -60,15 +59,6 @@ func (o options) SetLooseRackCheck(looseRackCheck bool) Options {
 	return o
 }
 
-func (o options) AcrossZones() bool {
```
Comment (xichen2020) [inline placement/options.go]: This is no longer supported?

Comment (xichen2020) [inline placement/options.go]: Discussed offline, this is and likely will never used and has become a maintenance burden so fine to remove

**Pull request merged**